runtime/hsa-runtime/core/driver/xdna/uapi/amdxdna_accel.h

```diff
@@ -6,7 +6,7 @@
 #ifndef AMDXDNA_ACCEL_H_
 #define AMDXDNA_ACCEL_H_
 
-#include <libdrm/drm.h>
```

Comment:
Why is this changing?

Reply:
This changed in the XRT shim: https://github.com/amd/xdna-driver/blob/main/src/include/uapi/drm_local/amdxdna_accel.h#L9. I just moved the changes over.

Reply:
Actually, this is a change I made when porting over this file. Since it is a user-mode API it really should be using libdrm, not the kernel headers which is what the xdna-driver version is pointing to.

Please keep this as a local change.

Reply:
Ah good to know I didn't realize that, I will change that.